app.js

```diff
@@ -241,17 +242,17 @@ const assistant = new Assistant({
       // parsed threadHistory to align with openai.responses api input format
       const parsedThreadHistory = threadHistory.join('\n');
 
-      const messages = `System: ${DEFAULT_SYSTEM_CONTENT}\n\n${parsedThreadHistory}\nUser: ${message.text}`;
-
       // Send message history and newest question to LLM
       const llmResponse = await openai.responses.create({
         model: 'gpt-4o-mini',
-        input: messages,
+        input: `System: ${DEFAULT_SYSTEM_CONTENT}\n\n${parsedThreadHistory}\nUser: ${message.text}`,
```

Comment:
note: Simplifying the code by removing the need for the variable `messages`. Personally, I found this variable name confusing and tried to find where else it was used (no where).